jumpscale/packages/admin/actors/admin.py

```diff
@@ -107,5 +107,17 @@ def delete_identity(self, identity_instance_name: str) -> str:
         else:
             return j.data.serializers.json.dumps({"data": f"{identity_instance_name} doesn't exist"})
 
+    @actor_method
+    def get_developer_options(self) -> str:
+        test_cert = j.core.config.get("TEST_CERT")
```

Comment:
we can add default value with false, in case users didn't make it in the first time ?

Reply:
it's set in frontend